src/main/kotlin/io/hemin/wien/model/Episode.kt

```diff
@@ -180,15 +188,16 @@ data class Episode(
             /** Type describing a trailer episode. */
             TRAILER("trailer");
 
-            companion object {
+            public companion object {
+
                 /**
                  * Factory method for the instance of the Enum matching the [type] parameter.
                  *
                  * @param type The string representation of the Enum instance.
                  * @return The Enum instance matching [type], or null if not matching instance exists.
                  */
-                fun of(type: String?): EpisodeType? = type?.let {
-                    values().find { t -> t.type == it.toLowerCase() }
+                public fun from(type: String?): EpisodeType? = type?.let {
+                    values().find { value -> value.type == it.toLowerCase() }
```

Comment:
When we rename the `of` method to `from` like in the Builders, we can also name the companion objects `Factory`, to be consistent

Reply:
Will do!